callflow/datastructures/supergraph.py

```diff
@@ -609,8 +610,8 @@ def add_callsites_and_modules_maps(self, module_callsite_map={}):
 
         # ----------------------------------------------------------------------
         LOGGER.info(
-            f'[{self.name}] Created ("module-to-callsite" = {len(self.callsite2module)}) '
-            f'and ("callsite-to-module" = {len(self.module2callsite)}) '
+            f'[{self.name}] Created ("module-to-callsite" = {len(self.module2callsite)}) '
```

Comment:
This was also a logging issue. Info about wrong objects were printed previously.